src/streams/ReadableStream.ts

```diff
@@ -37,7 +37,7 @@ export interface Source<T> {
 	 * @returns A promise that resolves when the source's start operation has finished.  If the promise rejects,
 	 * 		the stream will be errored.
 	 */
-	start(controller: ReadableStreamController<T>): Promise<void>;
+	start?: (controller: ReadableStreamController<T>) => Promise<void>;
```

Comment:
This is the same signature conversion that is not necessary and inconsistent with the rest of the code.  This one should be:
`start?(controller: ReadableStreamController<T>): Promise<void>;`